inference-engine/tests_deprecated/functional/shared_tests/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ list(APPEND SHARED_LIBRARIES
         ${NGRAPH_LIBRARIES}
         ie_tests
         ngraphFunctions
+        inference_engine_ir_readers
```

Comment:
Why? I wanted to avoid such dependency

Reply:
Some old tests depend on this component. I removed it.